draft-ietf-oauth-status-list.md

```diff
@@ -765,7 +794,12 @@ The Status List approaches these privacy implications by integrating the status
 
 The herd privacy is depending on the number of entities within the Status List called its size. A larger size results in better privacy but also impacts the performance as more data has to be transferred to read the Status List.
 
-Additionally, the Issuer may analyse data from the HTTP request to identify the Relying Party, e.g. through the sender's IP address. This behaviour may be mitigated by private relay protocols or other mechanism hiding the original sender like {{RFC9458}}.
+Additionally, the Issuer may analyse data from the HTTP request to identify the Relying Party, e.g. through the sender's IP address.
+
+This behaviour may be mitigated by:
+
+- private relay protocols or other mechanism hiding the original sender like {{RFC9458}}.
+- using Third Party Hosting, see [](#third-party-hosting).
```

Comment:
Not sure about this one, but I guess we are talking about a trusted third party?
```suggestion
- using trusted Third Party Hosting, see [](#third-party-hosting).
```